ui/apps/platform/src/Containers/ConfigManagement/Entity/Deployment/DeploymentFindings.tsx

```diff
@@ -21,10 +26,7 @@ const DeploymentFindings = ({ entityContext = {}, deploymentID }) => {
     );
 };
 
-DeploymentFindings.propTypes = {
-    entityContext: PropTypes.shape({}),
-    deploymentID: PropTypes.string.isRequired,
-};
+DeploymentFindings.propTypes = {};
```

Comment:
Is it safe to remove this? Doesn't seem necessary anymore.

Reply:
Yes! Good catch.